server/sonar-main/src/main/java/org/sonar/application/cluster/HazelcastCluster.java

```diff
@@ -43,27 +44,31 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.locks.Lock;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.sonar.application.AppStateListener;
+import org.sonar.cluster.ClusterObjectKeys;
+import org.sonar.cluster.localclient.HazelcastClient;
 import org.sonar.process.MessageException;
-import org.sonar.process.NodeType;
+import org.sonar.cluster.NodeType;
 import org.sonar.process.ProcessId;
 
 import static java.lang.String.format;
 import static java.util.stream.Collectors.toList;
+import static org.sonar.NetworkUtils.getHostname;
+import static org.sonar.NetworkUtils.getIPAddresses;
 import static org.sonar.application.cluster.ClusterProperties.HAZELCAST_CLUSTER_NAME;
-import static org.sonar.process.NetworkUtils.getHostname;
-import static org.sonar.process.NetworkUtils.getIPAddresses;
-import static org.sonar.process.cluster.ClusterObjectKeys.CLIENT_UUIDS;
-import static org.sonar.process.cluster.ClusterObjectKeys.CLUSTER_NAME;
-import static org.sonar.process.cluster.ClusterObjectKeys.HOSTNAME;
-import static org.sonar.process.cluster.ClusterObjectKeys.IP_ADDRESSES;
-import static org.sonar.process.cluster.ClusterObjectKeys.LEADER;
-import static org.sonar.process.cluster.ClusterObjectKeys.NODE_NAME;
-import static org.sonar.process.cluster.ClusterObjectKeys.NODE_TYPE;
-import static org.sonar.process.cluster.ClusterObjectKeys.OPERATIONAL_PROCESSES;
-import static org.sonar.process.cluster.ClusterObjectKeys.SONARQUBE_VERSION;
+import static org.sonar.cluster.ClusterObjectKeys.CLUSTER_NAME;
+import static org.sonar.cluster.ClusterObjectKeys.HOSTNAME;
+import static org.sonar.cluster.ClusterObjectKeys.IP_ADDRESSES;
+import static org.sonar.cluster.ClusterObjectKeys.LEADER;
+import static org.sonar.cluster.ClusterObjectKeys.LOCAL_MEMBER_UUIDS;
+import static org.sonar.cluster.ClusterObjectKeys.NODE_NAME;
+import static org.sonar.cluster.ClusterObjectKeys.NODE_TYPE;
+import static org.sonar.cluster.ClusterObjectKeys.OPERATIONAL_PROCESSES;
+import static org.sonar.cluster.ClusterObjectKeys.SONARQUBE_VERSION;
 
 public class HazelcastCluster implements AutoCloseable {
   private static Logger LOGGER = LoggerFactory.getLogger(HazelcastCluster.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make the "LOGGER" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "LOGGER" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3008)